main/lsp/requests/workspace_symbols.cc

```diff
@@ -16,15 +16,15 @@ namespace {
 
 class SymbolMatcher final {
 public:
-    static constexpr int MAX_RESULTS = 50;
-    static constexpr int MAX_LOCATIONS_PER_SYMBOL = 10;
+    static constexpr size_t MAX_RESULTS = 50;
```

Comment:
Sorry, 2 minor nits... can you add a comment above this to keep it in sync w/ the duplicated value in test_corpus?